### DISCUSSION TITLE
请问terrain中的200*200个数据点以上是什么意思？

### DISCUSSION THREAD

**Discussion by hyy-hyy-19309** [physics-data/faq#74] [Q&A] [answered]
是选择的区域范围只要覆盖过 `200*200` 就可以了中间可以跳，还是说用到的数据点必须是 `200*200` 以上的。感觉风向图用上 `200*200` 个数据点之后就会变得很密集了

**Comment by Harry-Chen** [comment, marked as answer]
至少是这么多数据点覆盖的区域，但是并不一定要这么多数据点，可以进行采样等操作。
